generate/generate.go

```diff
@@ -228,26 +228,56 @@ func CheckStack(root string, stack stack.S) ([]string, error) {
 		return nil, fmt.Errorf("checking for outdated code: %v", err)
 	}
 
+	logger.Trace().Msg("Listing current generated files.")
+
+	g, err := ListStackGenFiles(stack)
+	if err != nil {
+		return nil, fmt.Errorf("checking for outdated code: %v", err)
+	}
+	currentFiles := newStringSet(g...)
```

Comment:
The overall idea is to:

1 - List all current generated code
2 - As we check for outdated code, we remove files from the set of files
3 - In the end, if all files exist and are OK, the set will be empty, if not...we have some extraneous/old generated files hanging around